doc/performance-testing/cloud-run/0003_stress-test-january-2022.md

```diff
@@ -0,0 +1,90 @@
+# Stress Test Phase 3 January 2022
+
+Following successful Phase 2 Performance testing we need to run a stress test to identify the point of failure at 50 runner instances.
+
+## Runner settings
+
+| Setting | Value |
+| --- | ---| 
+| Concurrency | 115 |
+| Max instances   | 50 |
+| Min instances | 50 |
+| CPU | 4 |
+| Memory | 4G |
+| Version | 3.94.0 |
+
+
+## Benchmark settings
+
+| Setting | Value |
+| --- | ---| 
+| Clients per instance | 100 |
+| Clients hatch rate   | 50 |
+| Wait time minimum | 1 |
+| Wait time maximum | 2 |
+| Requests JSON | test_benchmark_business.json |
+| Runtime | 20m |
+
+## Initial Benchmark
+
+Ran an initial benchmark against 40 load injector instances to check that the results were comparable with the phase 2 test.
+
+The result was broadly comparable but slightly faster for the phase 3 test. 
+
+| Stress Test | Runner Concurrency |Runner Instances | Run Time| Clients per instance | Requests per second | 99th percentile Max CPU Usage (%) | 99th percentile response time (ms) | Error rate (%) | Output |
+| --- | --- | --- | --- |--- | --- | --- | --- | --- | --- |
+| Phase 2 | 115 | 50 | 20 mins | 100  | 4668 | 64.5 | 229 | 0.000 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2021-12-29T14:42:36) |
+| Phase 3 | 115 | 50 | 20 mins | 100  | 4668 |  61.5 | 167 | 0.000 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2022-01-28T10:26:34)|
+
+## Results
+
+| Load Injector Instances | Requests per second | 99th percentile Max CPU Usage (%) | 99th percentile response time (ms) | Error rate (%) | Output |
+| --- | --- | --- | --- | --- | --- |
+| 45  | 5240 | 70.5 | 166 | 0.000 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2022-01-28T11:12:11) |
```

Comment:
Am I correct in thinking, the run time for each of these were 20 mins?

Reply:
Yeah 20 mins 👍